### PR TITLE
[ticket/16812] login_body.html missing dt tag for autologin

### PR DESCRIPTION
**Pull request by Prosk8er**
PHPBB3-16812

Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [ ] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16812

### CODE REVIEW

File: phpBB/styles/prosilver/template/login_body.html [hunk 1]
```diff
@@ -27,6 +27,7 @@ <h2 class="login-title"><!-- IF LOGIN_EXPLAIN -->{LOGIN_EXPLAIN}<!-- ELSE -->{L_
 		<!-- ENDIF -->
 		<!-- IF S_DISPLAY_FULL_LOGIN -->
 		<dl>
+			<dt>&nbsp;</dt>
 			<!-- IF S_AUTOLOGIN_ENABLED --><dd><label for="autologin"><input type="checkbox" name="autologin" id="autologin" tabindex="4" /> {L_LOG_ME_IN}</label></dd><!-- ENDIF -->
 			<dd><label for="viewonline"><input type="checkbox" name="viewonline" id="viewonline" tabindex="5" /> {L_HIDE_ME}</label></dd>
```
Comment (3D-I) [inline phpBB/styles/prosilver/template/login_body.html:32]: For each `dd` a `dt` should be placed above it, AFAIR.

Comment (Crizz0) [reply phpBB/styles/prosilver/template/login_body.html:32]: No, there can be one dt with many dd elements.

**Review by hanakin — APPROVED** [review]
The main issue here is our horrible use of definition lists for layout. Not sure it's worth fixing this rubbish code over replacing it, but have at it. 😉

File: phpBB/styles/prosilver/template/login_body.html (hunk 1 above)
Comment (marc1706) [reply phpBB/styles/prosilver/template/login_body.html:32]: See also the specs: https://www.w3.org/TR/2011/WD-html5-author-20110809/the-dl-element.html

**Comment by hanakin** [comment]
@Prosk8er looks good you just need to redo your commit message to match the required format. https://area51.phpbb.com/docs/dev/3.3.x/development/git.html

**Comment by Crizz0** [comment]
Check https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by Prosk8er** [comment]
i was doing it directly from the browser on github so couldn't amend it
i now have it locally via git-scm which i should've done from the beginning i messed this one up and started a new one here https://github.com/phpbb/phpbb/pull/6241
i haven't done a pull request since 2013 or 2014 so forgot everything

**Pull request closed without merging**